src/components/groups/group-create.jsx

```diff
@@ -93,4 +94,4 @@ function GroupCreate() {
   );
 }
 
-export default GroupCreate;
+export default SideAndTopbarHOC(GroupCreate);
```

Comment:
Can this be added to another layer, so we do not have to do this for each page component?

Reply:
Yes, but I am not sure that would be the best way either, then all the files should go through _that_ layer, except one. Would you prefer this?